Tests/EndToEndTests/CNTKv2Python/Tutorials/CNTK_302B_Image_Super-resolution_Using_CNNs_and_GANs_test.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+# Licensed under the MIT license. See LICENSE.md file in the project root
+# for full license information.
+# ==============================================================================
+
+import os
+import re
+import numpy as np
+
+abs_path = os.path.dirname(os.path.abspath(__file__))
+notebook = os.path.join(abs_path, "..", "..", "..", "..", "Tutorials", "CNTK_302B_Image_Super-resolution_Using_CNNs_and_GANs.ipynb")
+datadir = os.path.join(abs_path, "..", "..", "..", "..", "Examples", "Image", "DataSets", "BerkeleySegmentationDataset")
+
+# Run this on GPU only
+notebook_deviceIdsToRun = [0]
+
+@pytest.fixture(scope='module')
+def clean_data(device_id):
+  # Delete the data folder if we're supposed to run
+  if device_id in notebook_deviceIdsToRun:
+    import subprocess
+    args = ["git", "clean", "-fdx", datadir]
+    subprocess.check_call(args)
+
+@pytest.mark.skipif(not reWeekly.search(os.environ.get('TEST_TAG')),
```

Comment:
we do want to run this more frequently than a weekly run. Please remove this tag.

Reply:
#@pytest.mark.skipif(not reWeekly.search(os.environ.get('TEST_TAG')),